src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -196,79 +199,101 @@ private SplitPane getPaneAndStartSearch(BibEntry entry) {
      * @param listView CheckListView to style
      */
     private void styleFetchedListView(CheckListView<CitationRelationItem> listView) {
-        PseudoClass entrySelected = PseudoClass.getPseudoClass("selected");
```

Comment:
Why the indent change? Please revert. Otherwise, this PR is not reviewable properly.

Reply:
I've fixed this too.